docs/manage/users-remote.rst

```diff
@@ -0,0 +1,117 @@
+.. _remote-users:
+
+#######################
+ Managing Remote Users
+#######################
+
+Determined lets you manage users and user provisioning remotely. Remote user provisioning lets you
+include and synchronize any information about the user stored in your IdP such as their username,
+groups, and display name. Once configured, you can manage remote users without having to manually
+modify the users or update them via SCIM. Each time the remote user accesses Determined, their
+information is synchronized.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+.. note::
+
+   Only OIDC is supported.
+
+*******************************
+ Enable Remote User Management
+*******************************
+
+Set the Auto Provision Option
+=============================
+
+The first step is to configure the :ref:`master configuration file <master-config-reference>` to
+enable auto provisioning users and the remote management of any information attached to the users.
+
+-  Set ``oidc.auto_provision_users`` option to ``true`` in your :ref:`master configuration file
+   <master-config-reference>`.
+
+.. note::
+
+   If ``scim_enabled`` is ``true``, then ``oidc.auto_provision_users`` must be ``false``.
+
```

Comment:
If SCIM is enabled (meaning that `scim.enabled: true` in the config), then this overrides any OIDC configuration. For best practice, set `oidc.auto_provision_users: false` in this case.

Reply:
resolved